tests/NgTests/Db2CatalogTests.cs

```diff
@@ -1161,6 +1354,8 @@ public async Task RunInternal_CallsCatalogStorageLoadStringExactlyOnce()
                 top: top,
                 verbose: true,
                 maxPageSize,
+                itemCacheControl,
+                finishedPageCacheControl,
```

Comment:
Consider changing to this for consistency throughout. 

```suggestion
                maxPageSize: Constants.MaxPageSize,
                itemCacheControl: Constants.NoStoreCacheControl,
                finishedPageCacheControl: Constants.NoStoreCacheControl,
```

Reply:
`maxPageSize: Constants.MaxPageSize` looks too verbose to me. The intent is obvious from the constant name, no need to make it even more explicit with parameter name. The other two though are a good clarification.

Reply:
Wait, the whole suggestion changes the code behavior: original code passes function arguments here, suggested change discards those arguments and passes constants.

Reply:
I left the code as-is. The variable names are descriptive enough.